src/components/DetailItems/NamespaceDetailItem.tsx

```diff
@@ -0,0 +1,44 @@
+import type { FC } from 'react';
+import { DetailsItem } from 'src/modules/Providers/utils/components/DetailsPage/DetailItem';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ResourceLink } from '@openshift-console/dynamic-plugin-sdk';
+import { getNamespace } from '@utils/crds/common/selectors';
+
+import type { ResourceDetailsItemProps } from './utils/types';
+
+const NamespaceDetailsItem: FC<ResourceDetailsItemProps> = ({
+  helpContent,
+  moreInfoLink,
+  resource,
+}) => {
+  const { t } = useForkliftTranslation();
+  const namespace = getNamespace(resource);
+
+  const defaultMoreInfoLink =
+    'https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces';
```

Comment:
Please create, if it does not exist, a central place for all links and move it there